lib/hooks/blueprints/actions/create.js

```diff
@@ -40,7 +40,7 @@ module.exports = function createRecord (req, res) {
 				Model.subscribe(req, newInstance);
 				Model.introduce(newInstance);
 			}
-			Model.publishCreate(newInstance.toJSON(), !req.options.mirror && req);
+			Model.publishCreate(newInstance, !req.options.mirror && req);
 		}
```

Comment:
Wow, i dont even remember. Actually it is kinda unneccessary, because the publishCreateSingle//former publishCreate  already does that.